wordcount/src/wordcount/Main.java

```diff
@@ -0,0 +1,59 @@
+
+		// theres 2 ways i can think of to do this. this is the first way
+		// the space in-between .split(" ") is needed to properly split out the un wanted characters.
+		System.out.println("looks like there are " + humanRights.split(" ").length + " words in the united nations declaration of human rights");
+
+		// way 2 is what you were really looking for though
+		// first we have to replace illegal or unwanted characters and split the words into an array
+		System.out.println("*** Lets do this the right way ***");
+		humanRights.replaceAll("[[\\.\\?\\!\\,\\;\\:\\{\\}\\(\\)\\']]", "");
+		String[] removedIllegals = humanRights.split(" +");
+
+		System.out.println("line 22 says there are " + removedIllegals.length + " words");
+
+		// now we need a hashmap and a for loop
+		// we need the hash map to store the array we made
+
+		HashMap<String, Integer> seprateHumanRightsHash = new HashMap<String, Integer>();
+
+		for (int i = 0; i < removedIllegals.length; i++){
+			if (seprateHumanRightsHash.containsKey(removedIllegals[i])){
+				seprateHumanRightsHash.put(removedIllegals[i],
+				seprateHumanRightsHash.get(removedIllegals[i]) + 1);
```

Comment:
When checking for existing words in your `HashMap`, make sure to treat words with different casing as the same item. An idea for this would be to use either `.toUpperCase()` or `toLowerCase()` every time you use the `.get` or `.put` methods so that the words will always have the same casing and thus be treated as the same `String` value.